services/src/main/java/org/keycloak/protocol/oidc/grants/ciba/channel/CIBAAuthenticationRequest.java

```diff
@@ -171,6 +174,14 @@ public ClientModel getClient() {
         return client;
     }
 
+    public String getClientNotificationToken() {
+        return clientNotificationToken;
+    }
+
+    public void setClientNotificationToken(String clientNotificationToken) {
+        this.clientNotificationToken = clientNotificationToken;
```

Comment:
Shall we enforce the `1024` character limit as per spec?

Reply:
I've added validation for this and also the test scenario. Thanks for pointing it!